README.md

```diff
@@ -162,14 +162,13 @@ Here is an example for using setup-cpp to make a builder image that has the Cpp
 FROM ubuntu:22.04 AS base
 
 # add setup-cpp
-WORKDIR "/"
-RUN apt-get update -qq
-RUN apt-get install -y --no-install-recommends wget
-RUN wget --no-verbose "https://github.com/aminya/setup-cpp/releases/download/v0.26.1/setup-cpp-x64-linux"
-RUN chmod +x ./setup-cpp-x64-linux
+RUN apt-get update && apt-get install -y \
+  npm \
+  && rm -rf /var/lib/apt/lists/*
```

Comment:
We don't need to clean the images here.

```suggestion
RUN apt-get update -qq
RUN apt-get install -y --no-install-recommends npm
```

Reply:
Yeah I understand, it's just [best practice](https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#run) but not really [necessary](https://github.com/moby/moby/blob/03e2923e42446dbb830c654d0eec323a0b4ef02a/contrib/mkimage/debootstrap#L82-L105).

Also [Multiple consecutive RUN instructions. Consider consolidation.](https://github.com/hadolint/hadolint/wiki/DL3059)

Reply:
> Also [Multiple consecutive RUN instructions. Consider consolidation.](https://github.com/hadolint/hadolint/wiki/DL3059)

Docker RUN commands are cached and are error safe, unlike Bash. So, I don't really agree with that.